src/core/storage/database/sqlite/SQLiteDatabase.ts

```
@@ -0,0 +1,81 @@
+	return sqlite({ locateFile: () => sqliteWasmUrl });
+};
+
+export class SQLiteDatabase implements SQLiteDB {
+	protected db;
+	constructor(data?: ArrayLike<number> | Buffer | null) {
+		this.db = loadSQLite().then((sqlite) => {
+			const db = new sqlite.Database(data);
+
+			// Notify updates
+			db.updateHook((operation, database, table, rowId) => {
+				this.onChangeCallbacks.forEach((callback) =>
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)